tools/recetox_msfinder/.shed.yml

```diff
@@ -0,0 +1,11 @@
+owner: recetox
+remote_repository_url: "https://github.com/RECETOX/galaxytools/tree/master/tools/recetox_msfinder"
+homepage_url: "https://github.com/RECETOX/recetox-msfinder"
+categories:
+  - Metabolomics
+repositories:
+  october_recetox_xmsannotator_advanced:
+    description: "Peak Annotation"
+    include:
+      - november_recetox_msfinder.xml
+      - macros.xml
```

Comment:
resolved in c28b5c1

Reply:
The shed.yml is only a metadata file for the ToolShed. Its not really taken into account during normal testing.

Reply:
If its not really needed, I would not use include, or exclude to avoid failures in the TS repo.

Reply:
The tools are also being pushed to the main toolshed by the CI, so I think all tools should have a .shed.yaml file to add the required metadata - I think otherwise it is also not possible to upload them there?